licensecheck.properties

```diff
@@ -0,0 +1,25 @@
+category=Languages
+name=License Check
+description=Checks Maven, Gradle and NPM project for license violations
+issueTrackerUrl=https://github.com/porscheinformatik/sonarqube-licensecheck/issues
+devVersion=5.0.0-beta-2
+publicVersions=4.0.2
+archivedVersions=3.2.0
+
+defaults.mavenGroupId=at.porscheinformatik.sonarqube.licensecheck
+defaults.mavenArtifactId=sonarqube-licensecheck-plugin
+
+5.0.0-beta-2.description=JavaScript-only and Groovy-only projects, configuration uses default functionality
+5.0.0-beta-2.sqVersions=[8.9,LATEST]
```

Comment:
As a 'devVersion' this version isn't publicly viewable. Thus it doesn't make sense to make it compatible with `LATEST`, especially since only one version can be compatible with `LATEST`.